.github/workflows/release.yaml

```diff
@@ -25,6 +25,76 @@ jobs:
           version: v3.12.0
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.5.0
+        id: chart_releaser
+        uses: helm/chart-releaser-action@v1.6.0
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      # Retrieve last tag pushed by cr
+      - name: Checkout
+        if: steps.chart_releaser.outputs.changed_charts != ''
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get Last Tag
+        id: last_tag
+        if: steps.chart_releaser.outputs.changed_charts != ''
+        run: echo "tag=$(git describe --abbrev=0 --tags)" >> "${GITHUB_OUTPUT}"
```

Comment:
Unfortunately, the new `chart_version` returned by helm/chart-releaser-action@v1.6.0 is the last chart version **before** the publication of the current one, we need a checkout to retrieve the tag created by chart-releaser.